test/fit_tests/deploy/rackhd_stack_init.py

```diff
@@ -138,7 +160,7 @@ def test06_discover_data_switch_node(self):
                          + str(api_data['status']))
 
```

Comment:
Another note for later. We need to add a check to make sure the switch was actually discovered correctly.